src/libs/UnreadIndicatorUpdater/updateUnread/index.ts

```diff
@@ -19,9 +18,12 @@ function updateUnread(totalCount) {
         // seems to improve this issue.
         document.title = '';
         document.title = hasUnread ? `(${totalCount}) ${CONFIG.SITE_TITLE}` : CONFIG.SITE_TITLE;
-        document.getElementById('favicon').href = hasUnread ? CONFIG.FAVICON.UNREAD : CONFIG.FAVICON.DEFAULT;
+        const favicon = document.getElementById('favicon') as HTMLLinkElement | null;
+        if (favicon?.href) {
```

Comment:
I would probably use instanceof here actually instead of assertion. WDYT? @kubabutkiewicz @fabioh8010

Reply:
```suggestion
        const favicon = document.getElementById('favicon');
        if (favicon instanceof HTMLLinkElement) {
```

Reply:
@BartoszGrajdek fixed !